.github/tests/integration/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ghcr.io/datachainlab/sgx-rust:2004-1.1.5
+
+SHELL ["/bin/bash", "-c", "-l"]
+
+WORKDIR /root
+
+RUN apt-get update -y && apt-get install jq make -y
```

Comment:
if the above comment is yes, do we need this line?

Reply:
Thanks.
This is unnecessary code and will be removed.